llama_stack/distribution/providers.py

```diff
@@ -31,6 +34,10 @@ def __init__(self, config, deps):
     async def initialize(self) -> None:
         pass
 
+    async def shutdown(self) -> None:
+        logger.debug("ProviderImpl.shutdown")
```

Comment:
why the debug?

Reply:
This seems to be the standard as far as I can tell, e.g. check the shudown funcs here https://github.com/meta-llama/llama-stack/blob/main/llama_stack/distribution/routers/routers.py

Reply:
OK. (We should eventually do that in a base class. Copying these over and over seems not optimal.)

Reply:
Agreed, though that may require some rework on the inheritance structure - all of these routers inherit from a unique subclass of the [Protocols](https://typing.python.org/en/latest/spec/protocol.html) type